dissect/target/plugins/os/windows/dpapi/master_key.py

```diff
@@ -86,6 +89,10 @@ def decrypt_with_hash_10(self, user_sid: str, password_hash: bytes) -> bool:
     def decrypt_with_password(self, user_sid: str, pwd: str) -> bool:
         """Decrypts the master key with the given user's password and SID."""
         for algo in ["sha1", "md4"]:
+            if algo not in hashlib.algorithms_available:
```

Comment:
Feel free to add a fallback to pycryptodome here, with the appropriate `HAS_CRYPTO` guard.